www/401.html

```diff
@@ -62,38 +84,43 @@ <h1 class="panel-title">401 - Unauthorized</h1>
               or check your <a href="https://id.apache.org/">Apache ID</a>.
             </p>
           </div>
-        </div>      
+        </div>
       </div>
-    </div>
-  </div>
-  
-  <div class="footer container-fluid">
-    <div class="row">
-      <div class="col-md-10 col-lg-10">
-        <div class="panel panel-default">
+      <div class="col-md-4">
+        <div class="panel panel-info">
           <div class="panel-heading">
-            <h3 class="panel-title">Related Apache Resources</h3>
+            <h3 class="panel-title">
+              Get Help!
+            </h3>
           </div>
           <div class="panel-body">
             <ul>
               <li>
-                <a href="https://infra.apache.org/">Apache Infra Team Projects</a>
+                <a href="mailto:dev@whimsical.apache.org?subject=[401] Not Authorized - help with:">Email dev@whimsical List</a>
               </li>
               <li>
-                <a href="https://projects.apache.org/">Other Apache Projects</a>
+                <a href="https://lists.apache.org/list.html?dev@whimsical.apache.org">Read our Mailing List</a>
               </li>
               <li>
-                <a href="https://reference.apache.org/">Apache Infra Reference Library</a>
-              </li>
-              <li>
-                <a href="https://www.apache.org/dev">Apache Developer How-Tos</a>
+                <a href="https://github.com/apache/whimsy/tree/master/www">See The Website Code</a>
               </li>
             </ul>
           </div>
         </div>
       </div>
     </div>
+    <div class="footer container-fluid" style="background-color: #f5f5f5; padding: 10px;">
+      <p class="center">
+        Copyright © 2017, the Apache Software Foundation. Licensed under the 
```

Comment:
:blush: :wave: D'oh, those files were meant to be on master.  Sorry!  Agreed in principle, but the error .html pages are pure static, so there's nothing to change.  This is the same footer/header used by other whimsy code.